test/setup/main.tf

```diff
@@ -64,6 +64,7 @@ module "forseti-host-project" {
     "sqladmin.googleapis.com",
     "logging.googleapis.com",
     "monitoring.googleapis.com",
+    "servicenetworking.googleapis.com",
```

Comment:
Does `main.tf` also need this API listed?

Reply:
I have updated the cloudsql module to handle this logic.

Reply:
Nevermind, I only fixed the shielded vm and shared vpc tests. The install simple test is done a little differently than would typically be deployed, so to handle that I am putting this service back in the test setup. I will leave [this](https://github.com/forseti-security/terraform-google-forseti/pull/581/files#diff-e68da704e977beb85b34e6adfa2bcebbR43) in which will handle the common scenarios where Forseti is deployed into a project that is different from where the network is setup. However, since the install simple tests are deployed into the same project as the network, it doesn't work there. To really fix it, we should change the install simple example test to be done where the network is setup in a different project.. and I don't have the heart to do that in this PR.

Reply:
@hshin-g Updated PTAL.